scripts/disabled_tests/tests/data/playlist_input.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<playlist>
+	<test>
+		<NewTestCaseName>ParallelStreamsLoadTest_special_J9</NewTestCaseName>
+	</test>
+	<test>
+		<a>
+			<b>
+				<c>
+					<testCaseName>ParallelStreamsLoadTest_special_J9</testCaseName>
+				</c>
+			</b>
+		</a>
+		<disables_new_name>
```

Comment:
I think it should be `<disables>` instead of `<disables_new_name>`

Reply:
The name is intentionally misspelled to show how the parser ignores the container name, and only searches for "disable" nodes.

Added a explanatory comment in fc4c5554c3371ed55a3b296079c4cb9a687ec007

Reply:
I think Renfei was looking at this from the TKG perspective, where `<disables_new_name>` is not allowed by the .[xsd](https://github.com/adoptium/TKG/blob/master/resources/playlist.xsd) file which is enforced by TKG, so while this parser would ignore parent container, the .xml files checked into the repo, it would only encounter them if someone manually changed the file (hopefully not a common case).  Same goes for the `<a><b><c>` examples... though I understand they are to describe generally how the script works.  (One could just use examples that are actually allowed by the schema).

Reply:
>  it would only encounter [the invalid .xml files] if someone manually changed the file

To me, the intent was more to show how various changes to the _official_ format (enforced by the .xsd) do or do not effect the parser (changes to the format that require an update to the parser or not). This intent to show how changes 
to the official format may require a change in the parser was my interpretation of Lan's advice:

> The idea is that we do not want this python parser to be duplicating the logic in TKG. Since the parser only has info about disables, it does not need to be maintained/updated for every playlist format change.
https://github.com/adoptium/aqa-tests/issues/3345#issuecomment-1036268620

---

> One could just use examples that are actually allowed by the schema

I can add more test files and include some as well if you would like. Perhaps:
- one formatted correctly (valid according to the .xsd)
- one formatted "fuzzy" (**not** valid according to the .xsd, but still accepted by the parser)
- and one formatted incorrectly (**not** valid according to the .xsd, and **not** accepted by the parser)

Reply:
You could increase the examples if you want... the main point is that if we give only the incorrect example, we risk 'confusing some people' who may think thats a big thumbs up to mis-format those files.  It may be sufficient to use a properly formatted one for the example and state that this script does not enforce the schema as TKG does (and reference the .xsd), so that even if you fed it incorrectly formatted files, it could still do its job.

Reply:
I see, yes it may be confusing, especially if we don't even have one "correct" xml test file.

I reverted the xml test file to a correct format and stated, in the readme, that the script does not rely on the xsd format to work. Please see 2157d9a3a0bc4d2a6692086882591e536fb0433e